libs/sdk/skyux-eslint/docs/rules/template/prefer-input-box.md

```
@@ -0,0 +1,129 @@
+
+<br>
+
+## Rule Options
+
+The rule does not have any configuration options.
+
+<br>
+
+## Usage Examples
+
+#### Default Config
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)